### ISSUE TITLE
Global defaults

### ISSUE THREAD

**Issue by rgvy** [1000hz/bootstrap-validator#572]
Hello

Here's a feature that I think many people would find useful: the ability to set global defaults. For example, in every single form in my website, I wish to to set validated forms with `data-disable="false" data-focus="false"`. Additionally, I would like to change the default error message for every required field from the terse "Fill out this field" to the friendlier "This field is required". Adding all these data attributes is bloating my HTML templates, and it'd be great to set these options once only and forget about it.

Thanks!

**Comment by cdhannes** [comment]
Absolutely! I updated from 0.9 to the latest version because the custom validators don't seem to work properly in the older version and I can't get rid of the focus scrolling. There are just too many forms to add it manually.
